### ISSUE TITLE
Contributing guide issue

### ISSUE THREAD

**Issue by danielkucera** [openshift/builder#151]
Hi,
following command returns an error:
https://github.com/openshift/builder/blame/master/CONTRIBUTING.md#L34

```
$ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true}]}}' --type=merge
The ClusterVersion "version" is invalid: spec.overrides.group: Required value
```

**Comment by danielkucera** [comment]
Is this the correct command?

```
oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true, "group": "apps/v1"}]}}' --type=merge
clusterversion.config.openshift.io/version patched
```

**Comment by gabemontero** [comment]
> Is this the correct command?
> 
> ```
> oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true, "group": "apps/v1"}]}}' --type=merge
> clusterversion.config.openshift.io/version patched
> ```

yes that is the correct command @danielkucera 

when @nalind mentioned this to us during our scrum this AM I did not look closely enough to scroll over and fully see the setting of group in one version but not the other.

that requirement came in a release or two ago, and the contributing guide was not updated to reflect that

would you happen to have a commit ready to create a PR with, or should someone of devex do it?

@openshift/openshift-team-developer-experience FYI

**Comment by nalind** [comment]
Fixed by #327.  Yay!
